js/handlers/buttonView.js

```diff
@@ -149,7 +149,7 @@ define([
         $button.removeClass("is-disabled").removeAttr("disabled");
         trickle._button._isDisabled = true;
         // move focus forward if it's on the aria-label
-        if (document.activeElement && document.activeElement.isSameNode(this.$('.aria-label')[0])) {
+        if (document.activeElement && document.activeElement.isSameNode && document.activeElement.isSameNode(this.$('.aria-label')[0])) {
```

Comment:
Under which conditions is `isSameNode` missing?

https://developer.mozilla.org/en-US/docs/Web/API/Node/isSameNode

Reply:
IE11 when within an iframe - which is specified in the issue.

If you want a bit more detail:
When there's no active element then `document.activeElement` evaluates to `null`
For some reason on IE11 - only when viewed within an iframe - it evaluates to a native [object] {} which passes the first check and then errors out when attempting to call `isSameNode`

Reply:
@oliverfoster when I was seeing this, it was upon resuming a course back into a page that contained trickle (in Saba LMS, IE11, course being rendered in iframe). Weirdly it didn't seem to happen if you went into that page from the menu. Unfortunately I didn't have time/budget to do any further investigation - as the course had no accessibility requirement it was easiest just to comment out the code... I can point you in the direction of the course in question and give you access to Saba if you want to do some further checking.

Reply:
Could you switch to an explicit test rather than an implied one for the `null` value?
```js
if (document.activeElement !== null && document.activeElement && document.activeElement.isSameNode(this.$('.aria-label')[0])) {
}
```

Reply:
I think there's been a misunderstanding. Your suggestion would still result in the same error. But if you want something more explicit I'll cook something up...

Reply:
Much better, ta.